tracer/src/Datadog.Trace/Util/EnvironmentHelpersNoLogging.cs

```diff
@@ -23,30 +22,32 @@ internal static bool IsServerlessEnvironment(out Exception? exceptionInReading)
         // Track the first exception encountered while reading env vars
         Exception? firstException = null;
 
-        var isServerless = CheckEnvVar(LambdaMetadata.FunctionNameEnvVar, ref firstException)
-                        || (CheckEnvVar(PlatformKeys.AzureAppService.SiteNameKey, ref firstException)
-                         && !CheckEnvVar(ConfigurationKeys.AzureAppService.AzureAppServicesContextKey, ref firstException))
-                        || (CheckEnvVar(PlatformKeys.GcpFunction.FunctionNameKey, ref firstException)
-                         && CheckEnvVar(PlatformKeys.GcpFunction.FunctionTargetKey, ref firstException))
-                        || (CheckEnvVar(PlatformKeys.GcpFunction.DeprecatedFunctionNameKey, ref firstException)
-                         && CheckEnvVar(PlatformKeys.GcpFunction.DeprecatedProjectKey, ref firstException));
-
+        var isServerless = TryCheckEnvVar(PlatformKeys.Aws.FunctionName, ref firstException)
+                        || (TryCheckEnvVar(PlatformKeys.AzureAppService.SiteNameKey, ref firstException)
+                         && !TryCheckEnvVar(ConfigurationKeys.AzureAppService.AzureAppServicesContextKey, ref firstException))
+                        || (TryCheckEnvVar(PlatformKeys.GcpFunction.FunctionNameKey, ref firstException)
+                         && TryCheckEnvVar(PlatformKeys.GcpFunction.FunctionTargetKey, ref firstException))
+                        || (TryCheckEnvVar(PlatformKeys.GcpFunction.DeprecatedFunctionNameKey, ref firstException)
+                         && TryCheckEnvVar(PlatformKeys.GcpFunction.DeprecatedProjectKey, ref firstException));
         exceptionInReading = firstException;
         return isServerless;
+    }
 
-        static bool CheckEnvVar(string key, ref Exception? storedException)
+    private static bool TryCheckEnvVar(string key, ref Exception? storedException)
```

Comment:
nit: we essentially have the same method in `EnvironmentHelpers` called `EnvironmentVariableExists` which IMO is a better name for it - can we use that here too?